src/api/general-services/backend-status.js

```diff
@@ -0,0 +1,72 @@
+const k8s = require('@kubernetes/client-node');
+const crypto = require('crypto');
+const config = require('../../config');
+
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const getBackendStatus = async (experimentId) => {
+  const response = {
+    worker: {
+      status: 'NotLaunched',
+      started: false,
+      ready: false,
+      restartCount: 0,
+    },
+  };
+
+  // This will not work in development as we are not in a cluster.
+  // Always say the worker is down.
```

Comment:
Why not always say that the worker is up in development instead? Since usually that is the case and the behaviour of the api when running tasks up to now was 'if in development assume the worker is up'.

Reply:
Good point, fixed.